src/main/scala/thurloe/crypto/Encryption.scala

```diff
@@ -0,0 +1,79 @@
+package thurloe.crypto
+
+import java.security.SecureRandom
+import javax.crypto.spec.{IvParameterSpec, SecretKeySpec}
+import javax.crypto.Cipher
+
+import org.apache.commons.codec.binary.Base64
+
+import scala.util.{Failure, Success, Try}
+
+sealed trait Encryption {
```

Comment:
Why bother w/ the trait if there's only one extender? Do you envision another extender?